docs/source/release_notes.rst

```diff
@@ -1,10 +1,49 @@
 .. ...........................................................................
-.. © Copyright IBM Corporation 2020                                          .
+.. © Copyright IBM Corporation 2021                                          .
 .. ...........................................................................
 
 Releases
 ========
 
+Version 1.3.0
+-------------
+Notes
+  * Change Ansible support from 2.0 to 2.9.
+  * smtctl: new module to enables/disable simultaneous MultiThreading mode.
+  * backup: Fix idempotency issues. Add new force option to overwrite a backup. Better examples.
+  * alt_disk: fix failure with no free disk available. issue #61.
+  * devices: Fix idempotency issues. Other issues: #59, #98.
+  * emgr: Fix idempotency issues.
+  * filesystem: Fix idempotency issues. Other issues: #76. Other improvements.
+  * lvg: Fix idempotency issues.
+  * lvm_facts: Display volume groups that are deactivated or varied off.
+  * lvol: Fix idempotency issues.Fix the wrong interpretation for attribute size (issue #72). Issue #100.
+  *  - Add strip_size attribute.
+  *  - Allow users to re-size (increase) logical volumes by using +<size><suffix>,
+  *    where suffix can be B/M/K/G or a bigger size value.
+  * nim: Add new attribute boot_client option to prevent nim from rebooting the client.
```

Comment:
Do we need to mention any issues that we identified and fixed?

Reply:
The nim boot_client option was added by a contribution from a customer that added the feature to prevent nim from rebooting the endpoints at the end.